core/routemgmt/getApi/getApi.js

```diff
@@ -91,7 +93,7 @@ function main(message) {
     gwInfo.gwUrl = message.gwUrlV2;
     gwInfo.gwAuth = message.accesstoken;
     // Obtain the API from the API GW
-    return utils2.getApis(gwInfo, message.spaceguid, message.basepath)
+    return utils2.getApis(gwInfo, message.spaceguid, message.basepath, message.limit, message.skip)
```

Comment:
If the values are undefined what’s the behavior of the api call?

Reply:
@rabbah The api gateway will ignore them. All apis will be returned.

Reply:
thanks.